.github/workflows/pr_push.yml

```diff
@@ -141,7 +141,14 @@ jobs:
       run: cmake --build build --target clang-format-check
 
     - name: Check CMake formatting
-      run: cmake --build build --target cmake-format-check
+      run: |
+        cmake --build build --target cmake-format-apply
+        if ! git diff | wc -l | grep -q "^0$"; then
+          echo "Errors found:"
+          git diff
+          exit 1  # Indicate failure
+        fi
+        cmake --build build --target cmake-format-check
```

Comment:
Is this line still needed? You exit on failure already.

Reply:
Done